scripts/export_lower_resolution.py

```diff
@@ -22,7 +22,8 @@ def filter_component(fs, segmentation, cochlea, seg_name, components):
     filter_mask = ~np.isin(segmentation, keep_label_ids)
     segmentation[filter_mask] = 0
 
-    # segmentation, _, _ = relabel_sequential(segmentation)
+    segmentation, _, _ = relabel_sequential(segmentation)
```

Comment:
I think that this is the culprit why IHC ids and synapse IDs don't match. We still need to be careful here though, if the max ID exceeds 65,... than we run out of uint16 IDs, which may happen for SGNs and is why I included this.

Maybe the best option is to remove the sequential relabeling here, and to raise a warning if we have larger values than the uint16 max id, as this won't lead to a big issue, just some ID clashes.